validators/UserValidator.go

```diff
@@ -0,0 +1,54 @@
+package validators
+
+import (
+	"errors"
+	"github.com/badoux/checkmail"
+	"github.com/jasongauvin/wikiPattern/models"
+	"strings"
+)
+
+func ValidateUser(user *models.User, action string) error {
+	switch strings.ToLower(action) {
+	case "update":
```

Comment:
Don't use a switch. `ValidateUser` should validate the entire user, and does not depend of it being updated or created or whatever. Also you user validator is not meant to also validate your login form, these are 2 completely different things